### ISSUE TITLE
Thorium signs me out of Google Account on closing.

### ISSUE THREAD

**Issue by pabloagn** [Alex313031/thorium#736]
**System Details**
 - OS: Windows 11
 - Thorium Version 124.0.6367.218 (Official Build) (64-bit)

**Problem**
Thorium signs me out of Google Account on closing every time.

**Additional Notes**
NA

**Stuff I've tried**
Already tried the solution mentioned in [this issue](https://github.com/Alex313031/thorium/issues/715) without luck.

Right now this makes Thorium basically unusable for me.

**Comment by gz83** [comment]
I believe these issues no longer exist in the new version.

**Comment by Defiect** [comment]
@gz83 I'm suddenly getting this issue again after updating to the M152 beta. The M151 beta wasn't giving me this issue at all.

Every time I close Thorium on my windows 11 machine, then reopen it, I see "Verify it's you" when I click on my profile icon at the top right, indicating that I've been signed out

**Comment by gz83** [comment]
@Defiect Maybe it's an exception? The M152 version has been released for some time, and I have only received your report for the time being.

**Comment by Defiect** [comment]
> [@Defiect](https://github.com/Defiect) Maybe it's an exception? The M152 version has been released for some time, and I have only received your report for the time being.

You're right, sorry about that! It was a fluke with Windows update, turns out. I tested M152 on my linux machine and it worked fine there. Thanks for keeping this project alive!
